### ISSUE TITLE
slideshow with different display-time

### ISSUE THREAD

**Issue by kaniggel** [derf/feh#554]
Hello!
I use feh to display a slide show on my Raspberry.

I pass 7sec as the time interval for the picture change. I have 5 photos in the picture collection that should change faster. (Display time only approx. 1 second). Therefore I had the idea to create a gif from the 5 jpgs. The gif is not shown animated on the Raspberry. Can I do that somehow? Or is it possible to define certain times for individual pictures?

I would be very happy about your help!

Thanks in advance!

**Comment by zuhataslan** [comment]
Yes, being able to set different times would be great! Maybe via exif data or maybe better to have like image1.jpg, image1.cfg in wich one sets different settings for just that one image, and everything else follows cli options and general settings.

Also different setting for images that have an .cfg in same dir or in a dedicated cfg dir. Would be awesome
